terraform_compliance/extensions/terraform.py

```diff
@@ -205,8 +205,7 @@ def _parse_configurations(self):
         resources = self.raw.get('configuration', {}).get('root_module', {}).get('resources', [])
 
         # Append module resources
-        for module in seek_key_in_dict(self.raw.get('configuration', {}).get('root_module', {}).get("module_calls", {}), "module"):
-            resources += module.get('module',{}).get("resources", [])
+        resources.extend(self.process_module_calls(self.raw.get('configuration', {}).get('root_module', {}).get("module_calls", {})))
```

Comment:
`self.resources` never gets set in this function (`_parse_configurations`), right? I am testing it with the plan in #498. While the resources are being extracted, they never make it into `self.resources`.

Reply:
No, its for configuration. `_parse_resources` is the one who initially parse resources.

Reply:
Okay. Discussion continued at #498 as well